multi/imxrt-multi/imxrt-multi.h

```diff
@@ -32,12 +40,62 @@ typedef struct {
 			unsigned int val;
 		} dir;
 	};
-} gpio_i_msg_t;
+} gpio_t;
+
+
+
+/* SPI */
+
+
+enum { msb = 0, lsb = 1};
+
+
+enum { spi_mode_0 = 0, spi_mode_1, spi_mode_2, spi_mode_3 };
 
 
 typedef struct {
+	enum { spi_config = 0, spi_transmit } type;
```

Comment:
spi_transmit implicates only sending information. IMHO "transaction" is more appropriate